developer/examples/build.vpc.boot.snapshot.pkr.hcl

```diff
@@ -0,0 +1,81 @@
+// packer {
+//   required_plugins {
+//     ibmcloud = {
+//       version = ">=v3.0.0"
+//       source = "github.com/IBM/ibmcloud"
+//     }
+//   }
+// }
+
+variable "IBM_API_KEY" {
+  type = string
```

Comment:
Hi Bhavesh, 

I ran make run-boot-snapshot and i am getting the attached error though there is no "vsi_boot_volume_id" specified in the example, can you please run this example and attach the result in the PR.

Thank you

Reply:
<img width="1728" alt="Screenshot 2023-06-04 at 1 44 07 PM" src="https://github.com/IBM/packer-plugin-ibmcloud/assets/127872893/43db8d6a-c3a0-46aa-b686-6ea9999656fe">

Reply:
hey, it is cause the latest changes were built at your end, please follow readme:

```
import go env...
make setup
make plugin
```